tools_and_services/stack_guideline.md

```diff
@@ -2,46 +2,82 @@
 ###### Or Ab-*stack*-tion!  🤦‍♂️
 ![abtion image](https://media.giphy.com/media/ZVik7pBtu9dNS/giphy.gif)
 
-
 In Abtion we strive for the easiest and most efficient workflow.
 This document is thought of as being a resource we can turn to if we get in doubt about how we deal with managing certain stuff, such as CI/CD.
 Also, to make it easier to on-board new employees.
 
+## Programming languages
 
-## Frontend
-- [React](https://reactjs.org/)
-- [React Native](https://reactnative.dev/)
+- JavaScript (TypeScript)
+- Ruby
+- C#
 
-#### Testing
-- [Jest](https://jestjs.io/)
-- [React Testing Library](https://testing-library.com/docs/react-testing-library/intro)
+## Managing DBs and runtimes
+- [asdf](https://asdf-vm.com/) to easily install the required runtimes for most of our projects
+- [docker-compose](https://docs.docker.com/compose/) to easily get the DBs started - using the right versions
 
-#### CI
-- [GitHub Actions](https://github.com/features/actions)
+Dotnet has it's own built-in sdk manager.
 
-#### CD
-- [Heroku](https://heroku.com/)
+## Frameworks
 
-#### Monitoring
-- [Rollbar](https://elements.heroku.com/addons/rollbar): Error-tracking as an add-on for Heroku
-- [Librato](https://elements.heroku.com/addons/librato): Platform monitoring as an add-on for Heroku
+### Web apps
 
-## Backend
-- [Ruby on Rails](https://rubyonrails.org/) with the [Muffi template](https://github.com/abtion/muffi)
-- [.NET](https://dotnet.microsoft.com/en-us/) with the [MuffiNet template](https://github.com/abtion/muffi.net)
+- [Ruby on Rails](https://rubyonrails.org/) using our [rails template](https://github.com/abtion/muffi)
+- [.NET](https://dotnet.microsoft.com/en-us/) using our [.NET template](https://github.com/abtion/muffi.net)
 
-#### Database
-- [PostgreSQL](https://www.postgresql.org/)
+### Websites
+- [Strapi](https://strapi.io/) as CMS
+- [Next.js](nextjs.org/) as frontend (with or without CMS)
+- [Wordpress](https://wordpress.com/)*
 
-#### CI
-- [GitHub Actions](https://github.com/features/actions)
+\* Wordpress has it's own section further down the page as it's a whole separate ecosystem
+
+### Mobile apps
+- [React Native](https://reactnative.dev/)
 
-#### CD
-- [Heroku](https://heroku.com/)
+## Tooling
 
-#### Preferred Heroku Add-ons
-— [Rollbar](https://elements.heroku.com/addons/rollbar)
-— [Heroku Postgres](https://elements.heroku.com/addons/heroku-postgresql)
+### Hosting
+- PaaS (Platform as a Service)
+  - [heroku](https://heroku.com)
+  - [scalingo](https://scalingo.com/)
```

Comment:
I think it'd make sense to put them in a preferred order with the European one first 🇪🇺

```suggestion
  - [scalingo](https://scalingo.com/)
  - [heroku](https://heroku.com)
```

Reply:
@hcarreras What is your opinion on this. It's not like we've taken a company wide decision to use scalingo over heroku.

The readme of our rails template is also very heroku-centric, and would have to be updated.

A quick workaround for now could also be to just add a parenthesis to the scalingo bullet point: `(european alternative)`. It would make it clear which role scalingo is playing for us atm.

Reply:
I misinterpreted Markus' comment. 
Heroku is still our default. Scalingo is still a test.